source/diagrams/manage-diagrams/create-diagrams.txt

```diff
@@ -10,4 +10,46 @@ Create Diagrams
    :depth: 1
    :class: singlecol
 
-Create diagrams.
\ No newline at end of file
+You can create new diagrams on the Mapping tab of Relational Migrator.
+You can also create multiple diagrams and navigate between them to view
+specific entities of your data model. By default, Relational Migrator
+creates the ``Main Diagram`` diagram with a Relational view and a
+MongoDB view.
+
+About this Task
+---------------
+
+Creating diagrams allows you to visualize tables, foreign key
+relationships, and data types in different formats. After you create a
+diagram, you can :ref:`clone <rm-clone-diagrams>`, :ref:`rename
+<rm-rename-diagrams>`, and :ref:`delete <rm-delete-diagrams>` your new
+diagram. 
+
+You must have at least one digram on your ribbon tab. 
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Click the plus sign button on the right of the workspace tab 
+   
+      .. image:: /img/diagrams/create_diagram.png
```

Comment:
**Suggestion**

I like the image here to center the user in the UI. Could you add a red square highlight on + button just for extra clarity? 

Something like:

![Screenshot 2024-01-10 at 1 41 47 PM](https://github.com/mongodb/docs-relational-migrator/assets/85948430/a747f99b-775b-4186-8253-1e902ab9449c)

Reply:
Done!